source/unified-test-format/tests/valid-pass/createEntities-operation.yml

```diff
@@ -0,0 +1,54 @@
+description: createEntities-operation
+
+schemaVersion: "1.8"
+
+createEntities:
+  - client:
+      id: client0
+  - database:
+      id: database0
+      client: client0
+      databaseName: database0
+  - collection:
+      id: collection0
+      database: database0
+      collectionName: collection0
+
+initialData:
+  - collectionName: collection0
+    databaseName: database0
+    documents: []
+
+tests:
+  - description: createEntities operation
+    operations:
+      - name: createEntities
+        object: testRunner
+        arguments:
+          entities:
+            - client:
+                id: client1
```

Comment:
Only benefit is if other spec authors use these as examples down the line. I have no objection to leaving this as-is.

Reply:
To be clear, I don't understand what benefit there is to using anchors for entity IDs in any test at all (not just this one).

Reply:
In the case of entity IDs where the anchor and value are almost always identical, I can see why it seems pointless. I think one small benefit is that a typo is more likely to be caught during JSON conversion instead of a runtime error when executing the tests (due to a failed entity lookup).

For database and collection names, I think the anchors serve a much greater purpose since the values are often very different from the anchor names.

Reply:
Gotcha. Catching typos does seem like a useful benefit.